netcam_rtsp.c

```diff
@@ -690,7 +702,7 @@ static int netcam_rtsp_resize(unsigned char *image , netcam_context_ptr netcam){
 
     retcd=my_image_copy_to_buffer(
          netcam->rtsp->swsframe_out
-        ,(uint8_t *)image
+        ,(uint8_t *)netcam->receiving->ptr
```

Comment:
This is causing a crash for me.  I think the image pointer has memory allocated to the size of the new image while the receiving pointer is sized to what the camera is sending.  To reproduce, set config parms to a bigger image than the camera.